CenterForOpenScience/osf.io
# [ENG-1417] Waffle so EGAP registrations are visible to admins

## Purpose

Adds a group and flag that give EGAP admins access to the EGAP Registration schema 

## Changes

- adds new manager method that allows users in the egap admins group in
- adds a management command that adds appropriately configures flags and groups
- removes some old waffling that was making it hard to maintain tests.

## QA Notes

Should be dev test only. But QA/admin should remember to remove the old ENABLE_INACTIVE_SCHEMA switch.

## Documentation

Docstrings, but no user facing docs.

## Side Effects

Removes old waffling for `ENABLE_INACTIVE_SCHEMAS` and changes some test behavior. You should be able to filter normally on registration schemas as you typically would.

## Migration
There is a migration in this ticket, but only for a waffle flag. If we get the ticket that handles waffle flags with a management command in before merging this, then remove this migration.

## Feature flag
'egap_admins'

## Ticket

https://openscience.atlassian.net/browse/ENG-1417
